src/js/page.js

```diff
@@ -367,7 +367,17 @@ export default {
             this._focusOnFirstQuestion( pageEl );
             this._toggleButtons( newIndex );
             pageEl.dispatchEvent( events.PageFlip() );
+            pageEl.setAttribute( 'tabIndex', 1 );
+            pageEl.focus();
```

Comment:
Did you look into why `_focusOnFirstQuestion` is not working?

Reply:
It seems that `_focusOnFirstQuestion` calls `trigger( 'fakefocus' );` .  After looking around for a while, it seems `fakefocus` is not handled anywhere.

Reply:
Well, it seems to me that focusOnFirstQuestion's purpose should be to focus on the first question... :)  So it makes sense to me to fix that instead. 

page.js should actually be de-jquerified (eventually). Listening should be done with `events.FakeFocus().type` and triggering with `events.FakeFocus()`. It looks like it's used to update the progress bar only.

Surprisingly this function only appears to be scrolling to the first question and not focusing at all. I don't remember why, but let's add the focus there! 

Note that a full-page form also doesn't focus on the first question. Is that also a problem with screen readers?

Reply:
moving the call to `.focus()` to the *'fakefocus'* event handler makes sense. however, I'm not sure why the event handler is not called at all in spite of the even being fired. this is happening for me locally.